images/rabbitmq-messaging-topology-operator/tests/main.tf

```diff
@@ -0,0 +1,33 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+resource "random_pet" "suffix" {}
+
+##resource "helm_release" "cert-manager" {
+#  name       = "cert-manager-${random_pet.suffix.id}"
+#  repository = "https://charts.jetstack.io"
+#  chart      = "cert-manager"
+#  timeout    = 600
+#
+#  namespace        = "cert-manager-${random_pet.suffix.id}"
+#  create_namespace = true
+#}
```

Comment:
Can you explain why this is commented out? If there's some bug we're waiting to have fixed, link it here. Otherwise we'll probably never get back to this, so we might as well just remove this rather than comment it out.

Reply:
deleted.
left over from testing